FreeRTOS/FreeRTOS
# Add async core yield test for SMP

Description
-----------
The test case tries to reproduce this issue
https://github.com/FreeRTOS/FreeRTOS-Kernel/issues/1086

In this PR:
* Add async core yield test for SMP to verify set core affinity implementation.

Test Steps
-----------
The unit test cases should has no error.

Checklist:
----------



Related Issue
-----------
https://github.com/FreeRTOS/FreeRTOS-Kernel/issues/1086


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
